container/release-cli.js

```diff
@@ -0,0 +1,139 @@
+const GitHub = require('github-api');
+const fs = require('fs');
+const readline = require('readline');
+const packageJson = require('./public/package.json');
+const color = require('cli-color');
+
+const github = new GitHub({
+  token: process.env.GITHUB_TOKEN
+});
+const repo = github.getRepo('SAP', 'luigi');
+
+const logWarning = str => console.log(color.yellow.bold(str));
+const logSuccess = str => console.log(color.green.bold(str));
+const logError = str => console.log(color.redBright.bold(str));
+
+async function getContainerReleases() {
```

Comment:
would be nice to have some small comment header on these functions. what the input and output looks like. easier to debug later 🤔